docs/error_codes.md

```diff
@@ -5,9 +5,10 @@
 - `GS001`: `Threshold needs to be defined`
 
 ### General gas related
-- `GS010`: `Not enough gas to execute safe transaction`
+- `GS010`: `Not enough gas to execute Safe transaction`
 - `GS011`: `Could not pay gas costs with ether`
 - `GS012`: `Could not pay gas costs with token`
+- `GS013`: `Safe transaction failed when gasPrice and safeTxGas where 0`
```

Comment:
Maybe it should be `GS002`

Reply:
Why? Is not init related

Reply:
hmm true ... it was also not really gas related ... so I was thinking maybe having a different code would make sense ... maybe `GS040`?

Reply:
I will leave it at `GS013` and rename the section to "gas/ execution"